src/components/offers.js

```diff
@@ -130,10 +168,14 @@ class Offers extends React.Component {
               <h2 style={{ textAlign: 'center' }}> Offers </h2>
 
               <ReactTable
-                defaultPageSize={10}
+                defaultPageSize={this.state.limit}
                 data={data}
                 columns={columns}
                 TheadComponent={TheadComponent}
+                onPageChange={e => this.changePage(e)}
+                onPageSizeChange={e => this.changeSize(e)}
+                pages={Math.ceil(countOffers / this.state.limit)}
```

Comment:
Check if the limit its set to 0. That might break this

Reply:
OK